gatherer.go

```diff
@@ -30,33 +30,31 @@ func (g *gatherer) Init(gen *generator.Generator) {
 }
 
 func (g *gatherer) Generate(f *generator.FileDescriptor) {
-	comments := make(map[string]string)
+	nameToNodeMeta := make(map[string]*NodeMeta)
 	for _, loc := range f.GetSourceCodeInfo().GetLocation() {
-		if loc.LeadingComments == nil {
-			continue
-		}
-
 		name, err := g.nameByPath(f.FileDescriptorProto, loc.Path)
 		if err != nil {
 			g.Debug("unable to convert path to name:", err.Error())
```

Comment:
I coded this to work with any entity that could have a leading comment not for any proto node. I'm not quite sure how should be the way to discriminate the nodes we're not interested in (for now) like `option`?